.github/workflows/command-fmt.yml

```diff
@@ -0,0 +1,52 @@
+name: Command-FMT
+
+on: 
+  workflow_dispatch: 
+    inputs:
+      pr:
+        description: Number of the Pull Request
+        required: true
+
+jobs:
+  cmd-fmt:
+    runs-on: ubuntu-latest
+    container:
+      image: paritytech/ci-unified:bullseye-1.75.0-2024-01-22-v20240109
+      ports:
+        - 80
+      volumes:
+        - my_docker_volume:/volume_mount
+    steps:
+      - name: Install gh cli
+        run: |
+          (type -p wget >/dev/null || (apt update && apt-get install wget -y)) \
+          && mkdir -p -m 755 /etc/apt/keyrings \
+          && wget -qO- https://cli.github.com/packages/githubcli-archive-keyring.gpg | tee /etc/apt/keyrings/githubcli-archive-keyring.gpg > /dev/null \
+          && chmod go+r /etc/apt/keyrings/githubcli-archive-keyring.gpg \
+          && echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+          && apt update \
+          && apt install gh -y
+          git config --global --add safe.directory '*'
+      - name: Download repo
+        uses: actions/checkout@v4
+      - run: gh pr checkout ${{ inputs.pr }}
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+      - name: Export branch name
+        run: echo "branch=$(git rev-parse --abbrev-ref HEAD)" >> "$GITHUB_OUTPUT"
+        id: branch
+      - name: Run FMT
+        run: |
+          # format toml.
+          # since paritytech/ci-unified:bullseye-1.73.0-2023-11-01-v20231204 includes taplo-cli
+          taplo format --config .config/taplo.toml
+      - name: Report failure
+        if: ${{ failure() }}
+        run: gh pr comment ${{ inputs.pr }} --body "Run for <code>fmt</code> failed. See logs <a href=\"$RUN\">here</a>."
```

Comment:
is there any advantage of gh cli over actions/github-script ?

Reply:
It's leaner and it produces more "human readable" errors.

Reply:
good to know, thnks!